tests/Form/Type/MediaTypeTest.php

```diff
@@ -123,6 +126,9 @@ public function testInvalidFormProviderOption(): void
         ]);
     }
 
+    /**
+     * @requires extension gd_info
```

Comment:
IMO the extension is `gd`:
```suggestion
     * @requires extension gd
```